option optimize_for = LITE_RUNTIME;

message NianTaskRewardRequest
{
 required uint32 task_id = 1;
 optional uint32 exchange_cnt = 2;
}
